christfellowshipchurch/web-app-v2
# Removed overlay for all default cards when there is no title

### About
Removed overlay for all default cards when there is no title

### Test Instructions
Open the testing link and go to /articles/cohorts-parenting-article
You will see that none of the cards have an overlay 

### Screenshots
<img width="1413" alt="Screen Shot 2023-05-09 at 12 07 29 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/46769629/a27b45bc-4c87-4a09-a493-4855f6d3654c">

### Closes Tickets
[CFDP-2545](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-2545)

### Related Tickets
N/A


[CFDP-2545]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2545?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
